assets/js/gateways/stripe.js

```
@@ -2,119 +2,180 @@
+              },
+            });
+
+            if (error) {
+              reject(error);
+            }
+          } catch(err) {
+            console.error('Payment Element error:', err);
+          }
+
+          resolve();
+        }));
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)